commands/speed.js

```diff
@@ -0,0 +1,67 @@
+const Discord = require('discord.js');
+/**
+ * 
+ * @param {Number} server 
+ * @param {Rcon} rcon 
+ * @param {Discord.Message} msg 
+ * @param {string} reason 
+ * @param {string} tojail
+*/
+async function runcommand(server, rcon, msg, newSpeed) {
+    if (!rcon.connected) {
+        await msg.channel.send(`S${server} is not connected the bot.`)
+        return;
+    }
+    //let res = await rcon.send(`/jail ${tojail} ${reason}`)
+    let res = await rcon.send(`/c game.speed = ${newSpeed}`)
+    if (!res) { // this command should not get a reply from the server. The new game speed should have printed in the map though.
```

Comment:
you could also do smt like  let speed = await rcon.send(`/c game.speed = ${newSpeed}`)
```suggestion
    let res = await rcon.send(`/c game.speed = ${newSpeed}`)
    let speed_check = await rcon.send(`/c rcon.print(game.speed)`)
    if(Number(speed_check) === newSpeed){
     // the 100% worked
    }
      if (!res) { // this command should not get a reply from the server. The new game speed should have printed in the map though.
```

Reply:
I'm sorry - I don't understand this change or the reason for it.

Reply:
you have to put everything that’s in if(!res) inside of it if(number(speed_check) === newspeed)

Reply:
I don't think we need speed check if we have the blow error catch.